src/app/auth/shared/validators/name.validator.ts

```diff
@@ -0,0 +1,5 @@
+import { FormControl } from '@angular/forms';
+
+export function nameValidator(control: FormControl): null | object {
+  return control.value.trim() ? null : { invalidName: `Enter correct name.` };
```

Comment:
Please change message to `Name cannot be empty.` (also you don't need template string here , you can use single quotes `'`)